lighthouse-core/config/config.js

```diff
@@ -193,13 +193,17 @@ function assertValidAudit(auditDefinition, auditPath) {
   if (typeof auditDefinition.meta.failureDescription !== 'string' &&
     auditDefinition.meta.informative !== true &&
     auditDefinition.meta.scoringMode !== Audit.SCORING_MODES.NUMERIC) {
-    log.warn('config', `${auditName} has no failureDescription and should.`);
+    throw new Error(`${auditName} has no failureDescription and should.`);
   }
 
   if (typeof auditDefinition.meta.helpText !== 'string') {
     throw new Error(
       `${auditName} has no meta.helpText property, or the property is not a string.`
     );
+  } else if (auditDefinition.meta.helpText === '') {
+    throw new Error(
```

Comment:
⬆️  YUP everything that he said.

i added 513c4bda to make sure our guidance on adding a custom audit also includes these guys. :)

I am gonna merge and we'll see if any custom audit authors balk. 😀